server.js

```diff
@@ -8,14 +8,10 @@ const passport = require('passport')
 const GoogleStrategy = require('passport-google-oauth').Strategy
 const admin = require('./routes/admin.js')
 const routes = require('./routes/index.js')
-const mongoose = require('mongoose');
-const flash = require('connect-flash');
+const flash    = require('connect-flash');
```

Comment:
I recommend avoiding additional space between identifier and `=` - it introduces a maintenance burden for updates that add to this section to maintain alignment (although it looks like this may have just been unintentional whitespace?).